astroquery/eso/core.py

```diff
@@ -207,6 +207,8 @@ def list_instruments(self, cache=True):
                     if instrument not in self._instrument_list:
                         self._instrument_list.append(instrument)
             self._instrument_list.append(u'harps')
+            self._instrument_list.append(u'ambient_paranal')
+            self._instrument_list.append(u'meteo_paranal')
```

Comment:
These three lines shouldn't be necessary, in principle.  Any idea why harps, ambient_paranal, and meteo_paranal don't show up [here](http://archive.eso.org/cms/eso-data/instrument-specific-query-forms.html)?